src/contexts/sendSearchRequest.ts

```diff
@@ -78,6 +86,7 @@ export const sendSearchRequest = async ({
                   responseLang: language,
                   maxSummarizedResults: summaryNumResults,
                   summarizerPromptName: summaryPromptName,
+                  promptText: summaryPromptText
```

Comment:
Does this work?
In other words, if summaryPromptText is just an empty string does it get ignored in the backend or would it fail? Perhaps instead check if it's empty and if so then just not add it here?